tests/test_sourmash.py

```diff
@@ -2522,8 +2522,8 @@ def test_gather_metagenome_traverse():
 
         assert 'found 12 matches total' in out
         assert 'the recovered matches hit 100.0% of the query' in out
-        assert '4.9 Mbp      33.2%  100.0%      NC_003198.1 Salmonella enterica subsp...' in out
-        assert '4.7 Mbp       0.5%    1.5%      NC_011294.1 Salmonella enterica subsp...' in out
+        assert '4.9 Mbp       33.2%  100.0%       0.0    NC_003198.1 Salmonella enterica subsp...' in out
```

Comment:
I think the abundance should be omitted when abundance isn't being used - seems less confusing.

Reply:
Fair point. The order that we put the % coverage and abundance made it so that it was simpler to include the whole section if you wanted to grab the name as well. I will break it up so that it does % coverage + name.

Reply:
To be clearer - in all cases where the avg_abund is 0.0 because either track abundance was off or ignore abundance was used, I think we should completely omit the column (& not display 0.0).  It should still be in the CSV output, tho.

Reply:
Gotcha.

Reply:
I shall go through and make those changes.

Reply:
@halexand Did you end up making these changes somewhere? I can go through now if not. cc: @ctb